Olfeo/olfeo-saas/ingest/parser.yml

```diff
@@ -0,0 +1,54 @@
+name: olfeo-saas
+ignored_values: ["-"]
+pipeline:
+  - name: json_livelog
+    external:
+      name: json.parse-json
+  - name: parsed_timestamp
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{json_livelog.message.timestamp}}'
+        output_field: timestamp
+        format: timestamp
+        timezone: UTC
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parsed_timestamp.timestamp}}"
+          observer.vendor: "Olfeo"
+          observer.type: "{{json_livelog.message.type}}"
+          observer.product: "Olfeo SAAS"
+          event.category: ["web"]
+          event.type: ["access"]
+          event.action: "{{json_livelog.message.action}}"
+          destination.ip: "{{json_livelog.message.dest_ip}}"
+          url.original: |
+            {%- if json_livelog.message.url != "" -%}
+              {{ json_livelog.message.url }}
+            {%- else -%}
+              {{ json_livelog.message.domain }}
+            {%- endif -%}
+          source.ip: "{{json_livelog.message.src_ip}}"
+          source.port: "{{json_livelog.message.src_port}}"
+          source.user.name: "{{json_livelog.message.display_name}}"
+          olfeo.request.category: "{{json_livelog.message.category_label}}"
+          olfeo.request.theme: "{{json_livelog.message.theme_label}}"
+          olfeo.request.user_id: "{{json_livelog.message.user_id}}"
```

Comment:
Why not using the ECS [`user.id`](https://www.elastic.co/guide/en/ecs/8.17/ecs-user.html#field-user-id) field?